ui/src/components/settings/sub_pages/profile/mod.rs

```diff
@@ -538,3 +564,22 @@ fn get_input_options(validation_options: Validation) -> Options {
         ..Options::default()
     }
 }
+
+fn get_status_option<'a>(cx: Scope<'a>, status: &IdentityStatus) -> (String, Element<'a>) {
+    let indicator = Status::from(*status);
+    (
+        serde_json::to_string::<IdentityStatus>(status).unwrap_or_default(),
+        cx.render(rsx!(div {
+                class: "settings-online-status",
+                Indicator {
+                    status: indicator,
+                    platform: Platform::Unknown
```

Comment:
Though only implemented in mock data (to my knowledge anyway - not sure if this changed since the last time it was ever used) you could probably passthrough the profile platform itself rather than leaving it to unknown.

Reply:
<img width="357" alt="Screenshot 2023-12-05 at 19 31 13" src="https://github.com/Satellite-im/Uplink/assets/34157027/2102c8d8-ee93-4215-a838-32df11d31e0b">

there are only these platforms and i simply choose the circle icon one since that one is the most generic one.
and should better reflect it being in settings than the others